packages/devtools_app/assets/scripts/inspector_polyfill_script.dart

```diff
@@ -143,88 +156,94 @@ String addServiceExtensions() {
+                    additionalJson['flexFactor'] = parentData.flex as int;
                     additionalJson['flexFit'] =
                         describeEnum(parentData.fit ?? FlexFit.tight);
                   }
                 }
               }
-              return additionalJson;
-            }),
-      );
-    }
-    return Future.value(<String, Object>{
+            }
+            return additionalJson;
+          }),
+    ) as Map<String, dynamic>;
+    return Future.value(<String, dynamic>{
       'result': result,
     });
   }
 
   Future<Map<String, dynamic>> setFlexFit(Map<String, String> parameters) {
-    final String id = parameters['id'];
-    final FlexFit flexFit =
-        toEnumEntry<FlexFit>(FlexFit.values, parameters['flexFit']);
+    final String id = parameters['id'] as String;
+    final parameter = parameters['flexFit'] as String;
+    final FlexFit flexFit = toEnumEntry<FlexFit>(FlexFit.values, parameter);
     final dynamic object = WidgetInspectorService.instance.toObject(id);
-    if (object == null) return null;
-    final render = object.renderObject;
-    final parentData = render.parentData;
     bool succeed = false;
-    if (parentData is FlexParentData) {
-      parentData.fit = flexFit;
-      render.markNeedsLayout();
-      succeed = true;
+    if (object != null) {
+      final render = object.renderObject;
+      final parentData = render.parentData;
+      if (parentData is FlexParentData) {
+        parentData.fit = flexFit;
+        render.markNeedsLayout();
+        succeed = true;
+      }
     }
     return Future.value(<String, Object>{
       'result': succeed,
     });
   }
 
   Future<Map<String, dynamic>> setFlexFactor(Map<String, String> parameters) {
-    final String id = parameters['id'];
-    final String flexFactor = parameters['flexFactor'];
-    final int factor = flexFactor == 'null' ? null : int.parse(flexFactor);
+    final String id = parameters['id'] as String;
+    final String flexFactor = parameters['flexFactor'] as String;
+    final factor = flexFactor == 'null' ? null : int.parse(flexFactor);
     final dynamic object = WidgetInspectorService.instance.toObject(id);
-    if (object == null) return null;
-    final render = object.renderObject;
-    final parentData = render.parentData;
     bool succeed = false;
-    if (parentData is FlexParentData) {
-      parentData.flex = factor;
-      render.markNeedsLayout();
-      succeed = true;
+    if (object != null) {
+      final render = object.renderObject;
+      final parentData = render.parentData;
+      if (parentData is FlexParentData) {
+        parentData.flex = factor;
+        render.markNeedsLayout();
+        succeed = true;
+      }
     }
     return Future.value({'result': succeed});
   }
 
   Future<Map<String, dynamic>> setFlexProperties(
       Map<String, String> parameters) {
-    final String id = parameters['id'];
+    final String id = parameters['id'] as String;
```

Comment:
Same question String and as String?

Reply:
parameters['id'] might be null. 
In null safe dart the ideal way to write this is
`final id = parameters['id']!;`
but you can't use `!` and make the code compatible with legacy dart.